src/modules/site-v2/base/utils/local_file.py

```
@@ -83,6 +84,9 @@ def remove(self):
       return True
     except FileNotFoundError:
       return False
+    except Exception as e:
```

We might need to expand some of the exceptions, e.g.: file permissions/file locks/etc, before returning `True`.

---

What do you think of [fd4192d](https://github.com/AndersenLab/CAENDR/pull/401/commits/fd4192d11f84f970839ea6e4653b115131cf7870)?

In the interest of not spending too much time on this -- I check if the file still exists, and suppress the error if it doesn't; otherwise, I propagate the error to be handled elsewhere.  It's not a watertight solution, but it should prevent a failed `remove` from flying under the radar...

---

sounds good.